src/components/main/SidebarFooter.vue

```diff
@@ -12,17 +29,97 @@
         <v-icon>mdi-help</v-icon>
       </v-btn>
     </v-col>
+    <v-dialog v-model="isVisible" max-width="600">
+      <v-card>
+        <v-card-title class="justify-center">
+          <span class="headline"
+            >進捗部屋の情報をcsv形式で入力してください</span
+          >
+        </v-card-title>
+        <v-card-text no-resize filled>
+          <v-textarea
+            v-model="inputData"
+            rows="20"
+            no-resize
+            filled
+            :rules="$rules.verifiedRoom"
+          />
+          <div v-show="showError" class="ErrorMessage" style="color: #ef5350">
+            データは6列で入力してください。
+          </div>
+        </v-card-text>
+        <v-card-actions>
+          <v-btn color="blue darken-1" text @click="hideModal"> close </v-btn>
+          <v-btn color="blue darken-1" text @click="saveData"> done </v-btn>
+        </v-card-actions>
+      </v-card>
+    </v-dialog>
   </v-row>
 </template>
 
 <script lang="ts">
-import Vue from 'vue'
-import { Component } from 'vue-property-decorator'
+import { parse } from 'csv-parse/browser/esm/sync'
+import { unparse } from 'papaparse'
 
-@Component
-export default class SidebarFooter extends Vue {
-  get version(): string {
-    return import.meta.env.VITE_APP_VERSION
-  }
+export default {
+  data() {
+    return {
+      inputData: '',
+      isVisible: false,
+      showError: false,
+      version: import.meta.env.VITE_APP_VERSION,
+    }
+  },
+  computed: {
+    isPrivilegedUser(): boolean {
+      return this.$store.direct.state.me?.privileged ?? false
+    },
+  },
+  methods: {
+    showModal() {
+      this.isVisible = true
+    },
+    hideModal() {
+      this.isVisible = false
+      this.showError = false
+      this.inputData = ''
+    },
+    isValidData(data: string): boolean {
+      const records = parse(data, {
+        delimiter: ',',
+        relax_column_count: true,
+      })
+      return records.every(record => record.length === 6)
+    },
+    async saveData() {
+      if (this.isValidData(this.inputData)) {
+        this.showError = false
+        try {
+          const csvData = unparse([[this.inputData]])
+          const response = await fetch('http://localhost:6006/api/rooms/all', {
+            //開発環境url
+            method: 'POST',
+            credentials: 'include',
+            headers: {
+              'Content-Type': 'text/csv',
+            },
+            body: csvData,
```

Comment:
```suggestion
            body: this.inputData,
```
で私の場合は正常にデータを送信できたのですが、そちらの環境でどのようなエラーが生じているか、また、devtoolsのネットワークタブを見てリクエストとレスポンスの内容がどのようになっていたかを教えていただきたいです。

Reply:
![スクリーンショット 2023-09-19 11 41 08](https://github.com/traPtitech/knoQ-UI/assets/93627045/19f82bf1-d7d9-4d52-b8f2-6e6088ad66b8)
ここの情報で十分でしょうか。this.inputDataが複数行に渡るデータの場合，400が返ってくる問題が起きています。

Reply:
ありがとうございます
できればペイロードの方も見たいです

Reply:
![スクリーンショット 2023-09-19 12 23 39](https://github.com/traPtitech/knoQ-UI/assets/93627045/a172091b-95a2-434f-9048-b2b94c381c07)
遅くなりすみません。こちらで大丈夫でしょうか。